File: week-06-Python/sentimental/mario/more/mario.py
from cs50 import get_int
height = 0
stop = False
while (not stop):
    height = get_int("Height: ")
    if height > 0 and height < 9:
        stop = True


def print_hash(i):
    for k in range(i, 0, -1):
        print("#", end='')


for i in range(1, height+1):
    for j in range(height-i, 0, -1):
        print(" ", end='')
    print_hash(i)
    print("  ", end='')
    print_hash(i)
    print()

# print_hash(height)

File: week-01-C/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int takeHeight(){
    int height = get_int("Height: ");
    int stop = 0;
    while(stop==0)
    {
        if(height>0 && height<9){
            stop=1;
        }
        else{
            int h = get_int("Height: ");
            height = h;
        }
    }
    return height;
}

void printHash(int i)
{
    for(int k=i; k>0; k--)
        {
            printf("#");
        }
}

void printPattern(int val)
{
    for(int i=1; i<=val; i++)
    {
        for(int j=val-i; j>0; j--)
        {
            printf(" ");
        }
        printHash(i);
        printf("  ");
        printHash(i);
        printf("\n");
    }
}

int main(void){
    int height = takeHeight();
    printPattern(height);
}
